.changeset/dirty-cheetahs-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': minor
```

Comment:
```suggestion
'@backstage/plugin-auth-backend': patch
```

Think we can consider this a non breaking change

Reply:
updated in https://github.com/backstage/backstage/pull/22561/commits/65424caa3a61825c4a919bd2b0278ff9f4bb33ec